### PR TITLE
Add fix for triple-equals

### PR DESCRIPTION
**Pull request by alexeagle**
#### PR checklist

- [ ] Addresses an existing issue: #0000
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### CHANGELOG.md entry:

[new-fixer] triple-equals

/cc @rkirov

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/tslint](https://github.com/palantir/tslint), @alexeagle! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: src/rules/tripleEqualsRule.ts [hunk 1]
```diff
@@ -74,9 +75,16 @@ function walk(ctx: Lint.WalkContext<Options>) {
             if ((node.operatorToken.kind === ts.SyntaxKind.EqualsEqualsToken ||
                  node.operatorToken.kind === ts.SyntaxKind.ExclamationEqualsToken) &&
                 !(isExpressionAllowed(node.right, ctx.options) || isExpressionAllowed(node.left, ctx.options))) {
-                ctx.addFailureAtNode(node.operatorToken, node.operatorToken.kind === ts.SyntaxKind.EqualsEqualsToken
-                                                         ? Rule.EQ_FAILURE_STRING
-                                                         : Rule.NEQ_FAILURE_STRING);
```
Comment (ajafff) [inline src/rules/tripleEqualsRule.ts]: You could just add this as the last parameter to the call:
```ts
Lint.Replacement.appendText(node.operatorToken.end, "=")
```

Comment (alexeagle) [inline src/rules/tripleEqualsRule.ts]: oh, clever, I like that better too

**Comment by JoshuaKGoldberg** [comment]
Perhaps there should be a separate category of "dangerous" fixes like this one or `function ()` to `() =>`?

**Comment by alexeagle** [comment]
We should discuss that in #2625 I think.

**Comment by giladgray** [comment]
@andy-hanson please update this branch so we can review it, or close it if not relevant.

**Comment by johnwiseheart** [comment]
Closing due to age and inactivity. Feel free to re-open or create a new pull request if you decide to continue working on this.

**Pull request closed without merging**